code/modules/cargo/packs/emergency.dm

```diff
@@ -82,7 +82,7 @@
 
 /datum/supply_pack/emergency/medemergencylite
 	name = "Emergency Medical Supplies (Lite)"
-	desc = "A less than optimal, but still effective, set of tools for emergency care. Contains a box of bodybags, some normal (and advanced) health analyzers, healing sprays, a single first aid kit, charcoal, some gauze, a bottle of toxins, and some spare medipens."
+	desc = "A less than optimal, but still effective set of tools for emergency care. Contains a box of bodybags, some normal (and advanced) health analyzers, healing sprays, a single first aid kit, charcoal, some gauze, a bottle of toxins, and some spare medipens." //Skyrat change, fixed grammatical error
```

Comment:
"A less than optimal, but still effective, set of tools for emergency care."
vs.
"A less than optimal, but still effective set of tools for emergency care."
the first is actually correct in terms of the comma. The comma is giving pause and showing that the little bit between the two commas are optional, a comment that is not needed.
"A less than optimal, but still effective, set of tools for emergency care."
vs.
"A less than optimal set of tools for emergency care."
 If you want to remove the second comma, remove the first as well.